src/Input/Input.js

```diff
@@ -280,6 +302,7 @@ export default class Input extends Component {
           onFocus={this.handleFocus}
           onChange={this.handleChange}
           disabled={disabled}
+          defaultValue={defaultValue}
```

Comment:
I don't think that we need to explicit the property, it's already inside the `other` variable when not destructured.